fsck.ocfs2/journal.c

```diff
@@ -1068,12 +1074,63 @@ static errcode_t ocfs2_clear_journal_flag(ocfs2_filesys *fs,
 	return ret;
 }
 
+static errcode_t ocfs2_clear_jbd2_errno(ocfs2_filesys *fs,
```

Comment:
I'd like rename it to "ocfs2_clear_journal_error".

Reply:
ok